components/Modal/ExportBlueprint.js

```diff
@@ -23,110 +39,111 @@ class ExportBlueprint extends React.Component {
 
   render() {
     return (
-      <div
-        className="modal fade"
-        id="cmpsr-modal-export"
-        ref={c => {
-          this.modal = c;
-        }}
-        tabIndex="-1"
-        role="dialog"
-        aria-labelledby="myModalLabel"
-        aria-hidden="true"
-      >
-        <div className="modal-dialog">
-          <div className="modal-content">
-            <div className="modal-header">
-              <button type="button" className="close" data-dismiss="modal">
-                <span className="pficon pficon-close" />
-              </button>
-              <h4 className="modal-title" id="myModalLabel">
-                <FormattedMessage defaultMessage="Export Blueprint" />
-              </h4>
-            </div>
-            <div className="modal-body">
-              <form className="form-horizontal">
-                <div className="form-group">
-                  <label className="col-sm-3 control-label" htmlFor="blueprint-name">
-                    <FormattedMessage defaultMessage="Blueprint" />
-                  </label>
-                  <div className="col-sm-9">
-                    <p className="form-control-static" id="blueprint-name">
-                      {this.props.blueprint}
-                    </p>
-                  </div>
+      <React.Fragment>
+        <a href="#" onClick={this.open}>
+          <FormattedMessage defaultMessage="Export Blueprint" />
+        </a>
+        <Modal show={this.state.showModal} onHide={this.close} id="cmpsr-modal-export">
+          <Modal.Header>
+            <Modal.CloseButton onClick={this.close} />
+            <Modal.Title>
```

Comment:
Oh, I missed this in my review just now...

Can you add an `id` here, similar to what's used in Manage Sources, like `id="title-export"?
And then add `aria-labelledby="title-export"` to the `<Modal ... >`?

These updates provide an accessible name for the modal for screen readers.

Reply:
@jgiardino thanks, fixed.

However these should all be done by patternly-react I believe.